tests/AheadOfTime/Trimming/check.ps1

```diff
@@ -40,7 +40,7 @@ function CheckTrim($root, $tfm, $outputfile, $expected_len) {
 # NOTE: Trimming now errors out on desktop TFMs, as shown below:
 # error NETSDK1124: Trimming assemblies requires .NET Core 3.0 or higher.
 $compressAllMetadata = $env:_kind -eq "-compressAllMetadata"
-$expectedForSelfContained = if ($compressAllMetadata) {300032} else {288256}
+$expectedForSelfContained = if ($compressAllMetadata) {288256} else {288256}
```

Comment:
What metadata does the compression refer to? If it's sigdata and optdata, does this flag make sense if they are removed during trimming, which the equal on/off sizes would indicate?

Reply:
Exactly.
Now that I removed rest of the metadata (incl. the nullness ones), I can remove this conditional logic alltogether and just keep a single size.

(will probably keep the two CI runs to catch for regressions when .e.g a a new resource is introduced)